google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditor.java

```diff
@@ -97,6 +97,8 @@ public AppEngineFlexibleDeploymentEditor(Project project, AppEngineDeployable de
     commonConfig.getDeployAllConfigsCheckbox().setSelected(false);
     commonConfig.getDeployAllConfigsCheckbox().setVisible(false);
 
+    addSettingsEditorListener(editor -> updateStagedArtifactNameEmptyText());
```

Comment:
since this listens to all changes in the editor, will we run into double calls to `updateStagedArtifactNameEmptyText` when its called on archive selector's doc listener?

Reply:
Ah good point, I just tested it and it looks like you're right, this handler is called even for changes to the artifact path field. I removed the line below to update as part of the document listener, PTAL.